test/loopback.test.js

```diff
@@ -608,4 +611,96 @@ describe('loopback', function() {
       expect(methodNames).to.include('prototype.instanceMethod');
     });
   });
+
+  describe('Remote method inheritance', function() {
+    function getAllMethodNamesWithoutClassName(Model) {
+      return Model.sharedClass.methods().map(function(m) {
+        return m.stringName.replace(/^[^.]+\./, ''); // drop the class name
+      });
+    }
+    var app;
+
+    beforeEach(function() {
```

Comment:
before/after hooks should be named, see https://github.com/strongloop/loopback-contributor-docs/blob/671a797623450fa94f5e3da05d12c34fc8cd5aa1/style-guide.md#layout-of-test-files